src/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -121,10 +122,10 @@ def ansible_playbook(environment, playbook, *cmd_args):
             return p.returncode
 
         def run_check():
-            return ansible_playbook(environment, args.playbook, '--check', *unknown_args)
+            return ansible_playbook(environment, args.playbook, args.use_factory_auth, '--check', *unknown_args)
```

Comment:
Hmm, it's kinda ugly just because of the current state of this function, but I'd actually suggest simply depending on the outer scope for the value of this variable, like we do with e.g. `public_vars` and a number of other outside variables used within `ansible_playbook`. Having a bunch of positional arguments here is a bit awkward I think.

Reply:
https://github.com/dimagi/commcare-cloud/pull/1702/commits/61df7271376d9381b3d5065ead051b6a6b1cd478